pages/hpc/data-transfer.md

```diff
@@ -1,5 +1,301 @@
+and you will have access to the same directories that you do when logged into any other CCR system.
+
+You can download single files through the browser but to take advantage of the benefits of Globus
+you will need to do endpoint to endpoint transfers. To do this, 
+you must configure a local endpoint in order to transfer files to/from
+your local computer. You can easily set up a Globus endpoint by
+installing [Globus Connect
+Personal](https://www.globus.org/globus-connect-personal) and setting up a personal endpoint.
+
+Using the web app, connect your local workstation endpoint with the
+CCR endpoint and transfer files easily using the Globus
+GUI.
+
+![](../images/globus-filemanager.png)
+
+
+### Guest Collections (Globus Shared Endpoints)
+
+Using a Guest Collection (also known as a "Shared Endpoint"), You can share
+any file or folder that you have access to with anyone who
+has a Globus account. This is particularly useful for external collaborations in which
+data sharing is necessary.  
+
+Detailed information on creating Guest Collections is available [at
+docs.globus.org](https://docs.globus.org/how-to/share-files/).
+
+
+### Secure Shell Copy
+
+Command line terminal access is provided via the SSH protocol, while command line file transfer 
+is available with sftp (Secure File Transfer Protocol) and scp (Secure Copy).
+The ssh, scp and sftp client software should be available on all Linux distributions 
+as part of the operating system installation.  
+
+*to/from the Frontends:*
+
+* This would be for simple file transfers that can complete within a 15 minute timeframe  
+
+*from the Compute Nodes:*
+
+* This would be for file transfers that can complete within a 72 hour window
+* This will only work for outbound transfers because compute nodes are not exposed externally
+* You must have an active job with an allocated compute node to get access to them
+
+!!! Note "SSH Keys required"
+    Users must use SSH keys to connect to CCR servers using SSH/SFTP/SCP.  Please [follow these instructions to upload your public SSH key](../portals/idm.md#ssh-keys) to the CCR identity management portal before attempting to connect to CCR's servers.  
+
+The Secure Copy utility, `scp`, can send data to and fetch data
+from a remote server.
+
+In the examples here, replace `<path-to-file>` with the path of the
+file you wish to copy, `<username>` with your CCR
+username, and `<target-path>` with the full path to the directory you
+would like to send the file to.
+
+```bash
+# Copying files from a local workstation to CCR Frontend Servers
+
+scp -i <path-to-yourSSHKey> <path-to-file> <username>@vortex.ccr.buffalo.edu.edu:<target-path>
+```
+
+```bash
+# Copying files from CCR Frontend Servers to a local workstation
+
+scp -i <path-to-yourSSHKey> <username>@vortex.ccr.buffalo.edu.edu:<path-to-file> <target-path>
```

Comment:
If you're on a CCR login node you shouldn't need to specify vortex.  I also wouldn't assume they're using SSH keys on their local machine so to keep it really simple:
```suggestion
scp <path-to-file> <local username>@your.computer.name:<target-path>
```